### DISCUSSION TITLE
fund.django-rest-framework.org outdated

### DISCUSSION THREAD

**Discussion by michael-k** [encode/django-rest-framework#7742] [Potential Issue]
I've noticed that the nav bars on [www.](https://www.django-rest-framework.org/) and [fund.](https://fund.django-rest-framework.org/topics/funding/) are different.

www:

![image](https://user-images.githubusercontent.com/152008/71985449-86cbc700-3222-11ea-9254-d02f599bb969.png)

fund:

![image](https://user-images.githubusercontent.com/152008/71985397-71569d00-3222-11ea-99ef-796d169bc447.png)

Problems with fund:
- `community` is missing
- `topics` contains some items that are now in `community`
  - Those items link to *www*, but result in 404.
- `next` and `previous` link to *www* pages, but result in 404 (items from *topics* that moved to *community*)
- `search` does not seem to work

**Comment by rpkilby** [comment]
I assume that what's happening here is that the funding site is just out of date, although that specific page has remained unchanged. 

@tomchristie maybe the funding site should just redirect to the www site?

**Comment by lovelydinosaur** [comment]
The funding site has a fixed copy of the header info, so yeah it needs updating.
It's just an extra drag that I've not gotten around to.

> maybe the funding site should just redirect to the www site?

The funding site itself needs to dynamically load the sponsor info, and handle payments, so we can't just redirect away for it.
